submissions/webdevtoday/html-css-popup/css/style.css

```diff
@@ -0,0 +1,203 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --icon-text-color: rgb(117, 117, 117);
+    --icon-outline-color: rgb(117, 117, 117);
+    --nav-popup-border-color: #e3e3e3;
+    --nav-popup-bg-color: #fff;
+    --nav-popup-height: 520px;
+}
+
+html {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+}
+
+ul li {
+    list-style: none;
+    display: block;
+}
+
+img {
+    display: block;
+    width: 100%;
+}
+
+.main-nav {
+    max-width: 1366px;
+    max-height: 80px;
+}
+
+.main-nav__container {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 12px 10px;
+}
+
+.main-nav__item {
+    position: relative;
+    display: block;
+    min-width: 32px;
+    min-height: 32px;
+    padding: 10px;
+    color: var(--icon-text-color);
+}
+.main-nav__item:hover,
+.main-nav__item:focus,
+.nav-popup__item:hover,
+.nav-popup__item:focus {
+    cursor: pointer;
```

Comment:
As I see this `cursor: pointer;` works on `focus` and `hover` by default, so maybe no need to add this style here

Reply:
Yes, you're right, I'll fix it.

Reply:
I've done it.